requirements.txt

```diff
@@ -2,3 +2,5 @@ Django==1.8.2
 django-extensions==1.5.5
 six==1.9.0
 django-bootstrap3
+djangorestframework
+djangorestframework-jwt
```

Comment:
Can we add specific versions to these requirements?  In the same form as a pip freeze command.